### ISSUE TITLE
Error on CreateAsync 

### ISSUE THREAD

**Issue by oesquedac** [wp-net/WordPressPCL#340]
Hi, i cant create a post using the version 2.0.1

This is the Code:

        var client = new WordPressClient( "http://sitioweb.com/wp-json/" );
        client.Auth.UseBasicAuth( "username", "app_password" );        

        var post = new Post() {
          Title = new Title( "ITCmx" ),
          Content = new Content( "Content PostCreate" )
        };

        var createdPost = await client.Posts.CreateAsync( post );

And get this error

Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'WordPressPCL.Models.Post' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.
To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.
Path '', line 1, position 1.

 I try on Postman and works fine

Could you please help me

Thank you

**Comment by Callightman** [comment]
Found any solution or workaround? Having same problem

**Comment by oesquedac** [comment]
Yes, is in the code around the path, you need to directly especify your url connection to your wordperfect api.

I mean:   private const string DEFAULT_PATH = "wp/v2/";

on the WordPressPCL,cs 

And is not in the documentation

**Comment by jduar005** [comment]
> Yes, is in the code around the path, you need to directly especify your url connection to your wordperfect api.
> 
> I mean: private const string DEFAULT_PATH = "wp/v2/";
> 
> on the WordPressPCL,cs
> 
> And is not in the documentation

Hello oesquedac, I appreciate your help here. I'm running into the same issue. Could you please elaborate on your workaround? 

I've tried specifying the full path to the API as "https://www.mydomain.com/wp-json/wp/v2" while setting the DEFAULT_PATH to an empty string.

And I've tried setting the path as "https://www.mydomain.com/wp-json" while explicitly specifying the DEFAULT_PATH = "wp/v2/".

This hasn't seemed to address the issue for me. I hope you're still around to help out!

Thank you.

**Comment by ThomasPe** [comment]
@jduar005 does it work through postman?

**Comment by oesquedac** [comment]
I resolve changing the global const DEFAULT_PATH 

Thank you

**Comment by mshasib** [comment]
Is there a workaround for this issue? Can someone please provide the details on how to fix this exactly?

**Comment by ThomasPe** [comment]
Closing because this report is stale and does not currently have a reproducible core-library defect. Please reopen with a minimal reproduction against the current release if the problem persists.
